src/runtime_src/core/tools/common/reports/platform/ReportRyzenPlatform.cpp

```diff
@@ -62,6 +62,16 @@ ReportRyzenPlatform::writeReport(const xrt_core::device* /*_pDevice*/,
       }
     }
 
+    const boost::property_tree::ptree& tops = pt_platform.get_child("tops", empty_ptree);
+    if (!tops.empty()) {
+      _output << std::endl << "TOPs" << std::endl;
```

Comment:
Please forget that `std::endl` ever existed.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#slio50-avoid-endl

Reply:
fixed.